src/desktop/package.json

```diff
@@ -122,9 +122,10 @@
       ]
     },
     "publish": {
-      "provider": "github",
+      "provider": "generic",
```

Comment:
I used a `generic` provider here since the binaries are simply being hosted here, but we don't want `electron-builder` to upload them. This part of the pipeline will do the uploading: https://github.com/iotaledger/ci-pipelines/blob/master/trinity-desktop-deploy/pipeline.yml#L13-L16

Reply:
Will the pipeline also create the required directory structure on s3?

Reply:
@geminoz will probably know the answer to this better than I do

Reply:
What's the required directory structure @rihardsgravis ? At present the pipeline uploads to an IF owned s3 bucket in a directory as defined by the CI tool. The s3 bucket is public but you cannot list the content of the bucket, meaning that in order to reach the files you have to know the exact full file path. The CI tool of course points you to the right artefact full path